### ISSUE TITLE
Enum fields show even for frameworks where they don't exist

### ISSUE THREAD

**Issue by svick** [dotnet/docs#2541]
In .Net Core 1.0, the `Method​Impl​Options.​Synchronized` enum field does not exist (it does exist in .Net Core 2.0 and in .Net Framework). [The page for this field correctly shows this.](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.methodimploptions.synchronized?view=netcore-1.0)

But [the `Method​Impl​Options` enum page](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.methodimploptions?view=netcore-1.0#fields_) does not: it lists `​Synchronized` among the fields even in the .Net Core 1.0 version of the page.

**Comment by mairaw** [comment]
The individual field pages have now been removed but the issue still continues on the main enum page. I couldn't find an internal bug filed on this.

@dend @adkinn FYI

These are the existing fields under .NET Core 1.0:
![image](https://user-images.githubusercontent.com/12971179/34634032-39148bfc-f235-11e7-87d2-be2e086754b7.png)

I also just noticed that enums don't have the "Applies to" section that was added to other APIs. So I'll be filing two bugs.
